tests/interfaces_test.py

```diff
@@ -294,29 +294,6 @@ def test_show_interfaces_portchannel_in_alias_mode(self):
         assert result.exit_code == 0
         assert result.output == show_interfaces_portchannel_in_alias_mode_output
         
-    @mock.patch('sonic_py_common.multi_asic.get_port_table', mock.MagicMock(return_value={}))
-    def test_supervisor_show_interfaces_alias_etp1_with_waring(self):
-        runner = CliRunner()
-        os.environ['SONIC_CLI_IFACE_MODE'] = "alias"
-        result = runner.invoke(show.cli.commands["interfaces"].commands["alias"], ["etp1"])
-        os.environ['SONIC_CLI_IFACE_MODE'] = "default"
-        print(result.exit_code)
-        print(result.output)
-        assert result.exit_code != 0
-        assert "Configuration database contains no ports" in result.output
```

Comment:
Instead of remove the whole test case, you may just remove the last assert.

Reply:
Done